ui/core/sim_manager.ts

```diff
@@ -79,7 +79,7 @@ export class SimManager {
 	 */
 	async abortAll(type: RequestType) {
 		for (const [id, cfg] of this.running) {
-			if (cfg.type != type) continue;
+			if ((cfg.type & type) == 0) continue;
```

Comment:
This should be `&&` and `===` in JS

Reply:
& is correct, they're bitmasks and the type can be multiple or all RequestTypes. I should probably rename those so it's clear.

Reply:
Ahhh ok! 👍